tests/JustSaying.UnitTests/Messaging/Channels/SubscriberBusTests/WhenPassingAHandledAndUnhandledMessage.cs

```diff
@@ -1,16 +1,34 @@
 using System;
+using System.Collections.Generic;
 using System.Threading;
 using Amazon.SQS.Model;
+using JustSaying.AwsTools.MessageHandling;
 using NSubstitute;
 using Xunit;
+using Xunit.Abstractions;
 
-namespace JustSaying.UnitTests.AwsTools.MessageHandling.SqsNotificationListener
+namespace JustSaying.UnitTests.Messaging.Channels.SubscriberBusTests
 {
-    public class WhenPassingAHandledAndUnhandledMessage : BaseQueuePollingTest
+    public class WhenPassingAHandledAndUnhandledMessage : BaseSubscriptionBusTests
```

Comment:
I don't understand the purpose and the naming of this test class. What am I missing?

Reply:
Agreed, I took all the existing SqsNotificationListener tests and made equivalent tests using the then SubscriptionBus so this one is copied from [here](https://github.com/justeat/JustSaying/blob/efae8047ef7ecbf540957a55794f22a54c18f341/tests/JustSaying.UnitTests/AwsTools/MessageHandling/SqsNotificationListener/WhenPassingAHandledAndUnhandledMessage.cs#L10)

Reply:
Will rename that folder/those tests to SubscriptionGroupTests now, but yeah I also don't understand this test. Happy to remove it if we don't think it has any value?